integration-tests/src/tests/ct_migration.rs

```diff
@@ -59,437 +48,134 @@ fn assert_migration_is_ready(project_id: u32) {
 	let mut inst = IntegrationInstantiator::new(None);
-	let project_id = PolitestNet::execute_with(|| {
+	PolitestNet::execute_with(|| {
 		let project_id = inst.create_finished_project(
 			default_project_metadata(0, ISSUER.into()),
 			ISSUER.into(),
 			default_evaluations(),
 			default_bids(),
 			default_community_contributions(),
-			vec![],
+			default_remainder_contributions(),
 		);
-
-		inst.advance_time(<PolitestRuntime as pallet_funding::Config>::SuccessToSettlementTime::get() + 1u32).unwrap();
+		inst.advance_time(<PolitestRuntime as pallet_funding::Config>::SuccessToSettlementTime::get()).unwrap();
+		inst.settle_project(project_id).unwrap();
 		project_id
-	});
-
-	mock_hrmp_establishment(project_id);
-
-	assert_migration_is_ready(project_id);
+	})
 }
 
 #[test]
-fn migration_is_sent() {
-	let mut inst = IntegrationInstantiator::new(None);
-	let participants =
-		vec![EVAL_1, EVAL_2, EVAL_3, BIDDER_1, BIDDER_2, BIDDER_3, BIDDER_4, BUYER_1, BUYER_2, BUYER_3, BUYER_4]
-			.into_iter()
-			.map(|x| AccountId::from(x))
-			.collect::<Vec<_>>();
-	let project_id = PolitestNet::execute_with(|| {
-		inst.create_finished_project(
-			default_project_metadata(0, ISSUER.into()),
-			ISSUER.into(),
-			default_evaluations(),
-			default_bids(),
-			default_community_contributions(),
-			default_remainder_contributions(),
-		)
-	});
-
-	PolitestNet::execute_with(|| {
-		dbg!(PolimecSystem::block_number());
-		dbg!(ProjectsToUpdate::<PolitestRuntime>::iter().collect::<Vec<_>>());
-	});
-
-	execute_cleaner(&mut inst);
+fn full_migration_test() {
+	let project_id = create_settled_project();
 
-	mock_hrmp_establishment(project_id);
-
-	assert_migration_is_ready(project_id);
-
-	send_migrations(project_id, participants);
-}
-
-#[test]
-fn migration_is_executed_on_project_and_confirmed_on_polimec() {
-	let mut inst = IntegrationInstantiator::new(None);
-	let participants =
-		vec![EVAL_1, EVAL_2, EVAL_3, BIDDER_2, BIDDER_3, BIDDER_4, BIDDER_5, BUYER_2, BUYER_3, BUYER_4, BUYER_5]
-			.into_iter()
-			.map(|x| AccountId::from(x))
-			.collect::<Vec<_>>();
-	let project_id = PolitestNet::execute_with(|| {
-		inst.create_finished_project(
-			default_project_metadata(0, ISSUER.into()),
-			ISSUER.into(),
-			default_evaluations(),
-			default_bids(),
-			default_community_contributions(),
-			default_remainder_contributions(),
-		)
-	});
-	execute_cleaner(&mut inst);
-
-	mock_hrmp_establishment(project_id);
-
-	assert_migration_is_ready(project_id);
-
-	let migrations_map = send_migrations(project_id, participants);
-	let grouped_migrations = migrations_map.values().cloned().collect::<Vec<_>>();
-
-	migrations_are_executed(grouped_migrations.clone());
-
-	migrations_are_confirmed(project_id, grouped_migrations.clone());
-}
-
-#[test]
-fn vesting_over_several_blocks_on_project() {
-	let mut inst = IntegrationInstantiator::new(None);
 	let participants = vec![EVAL_1, EVAL_2, EVAL_3, BIDDER_1, BIDDER_2, BUYER_1, BUYER_2]
```

Comment:
let's have some sort of runtime check that all participants defined here are the ones that participated, or retrieve them from the migrations storage item